### PR TITLE
New: Nested Tabs - Style icon section [ED-7334]

### PR DESCRIPTION
**Pull request by hein-obox**

### CODE REVIEW

File: modules/tabs-v2/assets/scss/frontend.scss [hunk 1]
```diff
@@ -100,18 +110,51 @@
 			text-align: var( --tabs-v2-title-alignment );
 			flex-grow: var( --tabs-v2-title-grow );
 
+			&:hover {
+
+				.elementor-tab-icon{
```
Comment (udidol) [inline modules/tabs-v2/assets/scss/frontend.scss:115]:
```suggestion
				.elementor-tab-icon {
```

File: modules/tabs-v2/assets/scss/frontend.scss [hunk 2]
```diff
@@ -100,18 +110,51 @@
...
+
+					>i {
```
Comment (udidol) [inline modules/tabs-v2/assets/scss/frontend.scss:117]:
```suggestion
					> i {
```

File: modules/tabs-v2/assets/scss/frontend.scss [hunk 3]
```diff
@@ -100,18 +110,51 @@
...
 				>i {
-					width: 20px;
+					color: var( --tabs-v2-icon-color );
+				}
+
+				svg {
+					fill: var( --tabs-v2-icon-color );
+				}
+	
+				&, &-active {
```
Comment (udidol) [inline modules/tabs-v2/assets/scss/frontend.scss:137]:
```suggestion
				&,
				&-active {
```

File: modules/tabs-v2/assets/scss/frontend.scss [hunk 4]
```diff
@@ -100,18 +110,51 @@
...
+
+					>i {
```
Comment (udidol) [inline modules/tabs-v2/assets/scss/frontend.scss:139]:
```suggestion
					> i {
```

File: modules/tabs-v2/assets/scss/frontend.scss [hunk 5]
```diff
@@ -100,18 +110,51 @@
...
+					svg {
+						fill: var( --tabs-v2-icon-color-hover );
+					}
+				}
+			}
+
 			.elementor-tab-icon {
-				display: unset;
-	
+
```
Comment (udidol) [inline modules/tabs-v2/assets/scss/frontend.scss:128]: Please add a space below here as well (`> i`)

File: modules/tabs-v2/assets/scss/frontend.scss [hunk 6]
```diff
@@ -100,18 +110,51 @@
...
+					svg {
+						width: var( --tabs-v2-icon-size );
+						height: var( --tabs-v2-icon-size );
+					}
 				}
 	
 				&-active {
 					display: none;
-	
+
```
Comment (udidol) [inline modules/tabs-v2/assets/scss/frontend.scss:151]: Same under this line

File: modules/tabs-v2/widgets/tabs-v2.php [hunk 7]
```diff
@@ -523,6 +526,152 @@ protected function register_controls() {
...
+				'bottom' => [
+					'title' => esc_html__( 'Bottom', 'elementor' ),
+					'icon' => 'eicon-v-align-bottom',
+				],
+				'start' => [
+					'title' => esc_html__( 'Left', 'elementor' ),
+					'icon' => 'eicon-h-align-' . $start,
+				],
+			],
+			'separator' => 'before',
```
Comment (udidol) [inline modules/tabs-v2/widgets/tabs-v2.php:556]: Why the separator?

File: modules/tabs-v2/widgets/tabs-v2.php [hunk 8]
```diff
@@ -523,6 +526,152 @@ protected function register_controls() {
...
+					'title' => esc_html__( 'Right', 'elementor' ),
+					'icon' => 'eicon-h-align-' . $end,
+				],
+				'bottom' => [
+					'title' => esc_html__( 'Bottom', 'elementor' ),
+					'icon' => 'eicon-v-align-bottom',
+				],
+				'start' => [
+					'title' => esc_html__( 'Left', 'elementor' ),
+					'icon' => 'eicon-h-align-' . $start,
```
Comment (udidol) [inline modules/tabs-v2/widgets/tabs-v2.php:553]: Maybe try using `eicon-order-start`/`eicon-order-end`?
See: https://elementor.github.io/elementor-icons/

**Review by udidol — CHANGES_REQUESTED** [review]

File: modules/tabs-v2/widgets/tabs-v2.php (hunk 7 above)
Comment (hein-obox) [reply modules/tabs-v2/widgets/tabs-v2.php]: @udidol removed

File: modules/tabs-v2/assets/scss/frontend.scss (hunk 1 above)
Comment (hein-obox) [reply modules/tabs-v2/assets/scss/frontend.scss]: updated

File: modules/tabs-v2/assets/scss/frontend.scss (hunk 3 above)
Comment (hein-obox) [reply modules/tabs-v2/assets/scss/frontend.scss:137]: updated - after next push

File: modules/tabs-v2/assets/scss/frontend.scss (hunk 4 above)
Comment (hein-obox) [reply modules/tabs-v2/assets/scss/frontend.scss]: updated

File: modules/tabs-v2/assets/scss/frontend.scss [hunk 9]
```diff
@@ -49,7 +51,10 @@
 	--tabs-v2-title-shadow: none;
 
 	--tabs-v2-icon-color: var( --e-global-color-secondary, var( --tabs-v2-color-secondary-fallback ) );
-	--tabs-v2-icon-active-color: #{$white};
+	--tabs-v2-icon-color-hover: var( --tabs-v2-title-color-hover );
+	--tabs-v2-icon-color-active: #{$white};
+	--tabs-v2-icon-size: initial;
```
Comment (udidol) [inline modules/tabs-v2/assets/scss/frontend.scss:56]: The `.elementor-tab-icon` element should receive a `display: flex` property to make sure the icon is vertically centered when it is smaller than the title text.

**Review by udidol — CHANGES_REQUESTED** [review]

File: modules/tabs-v2/assets/scss/frontend.scss (hunk 9 above)
Comment (hein-obox) [reply modules/tabs-v2/assets/scss/frontend.scss:56]: thanks!

updated without css variable > I don't think there is any need for a variable here

image

Comment (udidol) [reply modules/tabs-v2/assets/scss/frontend.scss:56]: Why do you need the display: flex both in the main title selector and the -active selector?

File: modules/tabs-v2/widgets/tabs-v2.php [hunk 10]
```diff
@@ -661,7 +663,7 @@ protected function register_controls() {
 			'global' => [
 				'default' => Global_Typography::TYPOGRAPHY_ACCENT,
 			],
-			'selector' => '{{WRAPPER}} .elementor-tab-title',
+			'selector' => '{{WRAPPER}} .e-tab-title-text',
 		] );
```
Comment (hein-obox) [inline modules/tabs-v2/widgets/tabs-v2.php:667]: I noticed that the typography (underline) was affecting the icon. I have targetted it more specifically now.

**Review by udidol — APPROVED** [review]

**Pull request merged**